drivers/include/l3g4200d.h

```diff
@@ -76,45 +76,35 @@ typedef enum {
 } l3g4200d_mode_t;
 
 /**
- * @brief   Device descriptor for L3G4200D sensors
+ * @brief   Device initialization parameters
  */
 typedef struct {
     i2c_t i2c;              /**< I2C device the sensor is connected to */
     uint8_t addr;           /**< the sensors slave address on the I2C bus */
     gpio_t int1;            /**< INT1 pin */
     gpio_t int2;            /**< INT2 (DRDY) pin */
-    int32_t scale;          /**< scaling factor to normalize results */
-} l3g4200d_t;
+    l3g4200d_mode_t mode;   /**< sampling frequency and bandwidth mode */
+    l3g4200d_scale_t scale; /**< scaling factor to normalize results */
+} l3g4200d_params_t;
```

Comment:
I'm not sure these and the other struct changes belong to this PR. Can you explain them?